src/middleware.js

```diff
@@ -158,6 +162,12 @@ const getLinks = payload => {
   };
 };
 
+function saveMeta(action, dispatch) {
```

Comment:
Move it with getMeta to below ln154 so they are organized together by name not by function get/set.